apstools/plans/alignment.py

```diff
@@ -16,6 +16,9 @@
 
 import numpy as np
 import pyRestTable
+from scipy.optimize import curve_fit
```

Comment:
If scipy is a new project requirement, then it should be added to `requirements.txt` and `environment.yml`.  Hold on this until PR #951 is merged into main and main merged into this.  It changes the packaging files.

Reply:
@MDecarabas - PR #951 has been merged into main.  Merge main into this branch.

Reply:
You mean rebase?

Reply:
Since I never use rebase, it's up to you.

Reply:
> If scipy is a new project requirement, then it should be added

```bash
(base) prjemian@arf:~/.../BCDA-APS/apstools$ git grep scipy
apstools/plans/alignment.py:from scipy.optimize import curve_fit
apstools/plans/alignment.py:from scipy.special import erf
```

Needs to be added to both `pyproject.toml` and `scipy`.

Reply:
The main branch has been switched to new Python packaging model.  `setup.cfg` is replaced by `pyproject.toml` in #951.  That makes a case for you to bring the new changes in main into this branch (merge or rebase, your choice).